packages/react-core/src/components/Page/__tests__/Page.test.tsx

```diff
@@ -80,6 +84,151 @@ test('Check page to verify breadcrumb is created', () => {
   expect(view.find('.pf-c-page__main').getDOMNode().id).toBe('');
 });
 
+test('Check page to verify breadcrumb is created - PageBreadcrumb syntax', () => {
+  const Header = <PageHeader logo="Logo" headerTools="PageHeaderTools | Avatar" topNav="Navigation" />;
+  const Sidebar = <PageSidebar isNavOpen />;
+
+  const view = mount(
+    <Page {...props} header={Header} sidebar={Sidebar}>
+      <PageBreadcrumb>
+        <Breadcrumb>
+          <BreadcrumbItem>Section Home</BreadcrumbItem>
+          <BreadcrumbItem to="#">Section Title</BreadcrumbItem>
+          <BreadcrumbItem to="#">Section Title</BreadcrumbItem>
+          <BreadcrumbItem to="#" isActive>
+            Section Landing
+          </BreadcrumbItem>
+        </Breadcrumb>
+      </PageBreadcrumb>
+      <PageSection variant="default">Section with default background</PageSection>
+      <PageSection variant="light">Section with light background</PageSection>
+      <PageSection variant="dark">Section with dark background</PageSection>
+      <PageSection variant="darker">Section with darker background</PageSection>
+    </Page>
+  );
+  expect(view).toMatchSnapshot();
+  expect(view.find('.pf-c-page__main').getDOMNode().id).toBe('');
+});
+
+test('Check page to verify nav is created - PageNavigation syntax', () => {
+  const Header = <PageHeader logo="Logo" headerTools="PageHeaderTools | Avatar" topNav="Navigation" />;
+  const Sidebar = <PageSidebar isNavOpen />;
+
+  const view = mount(
+    <Page {...props} header={Header} sidebar={Sidebar}>
+      <PageNavigation>
+        <Nav aria-label="Nav" variant="tertiary">
+          <NavList>
+            <NavItem itemId={0} isActive>
+              System Panel
+            </NavItem>
+            <NavItem itemId={1}>Policy</NavItem>
+            <NavItem itemId={2}>Authentication</NavItem>
+            <NavItem itemId={3}>Network Services</NavItem>
+            <NavItem itemId={4}>Server</NavItem>
+          </NavList>
+        </Nav>
+      </PageNavigation>
+      <PageSection variant="default">Section with default background</PageSection>
+      <PageSection variant="light">Section with light background</PageSection>
+      <PageSection variant="dark">Section with dark background</PageSection>
+      <PageSection variant="darker">Section with darker background</PageSection>
+    </Page>
+  );
+  expect(view).toMatchSnapshot();
+  expect(view.find('.pf-c-page__main').getDOMNode().id).toBe('');
+});
+
+test('Check page to verify grouped nav and breadcrumb - new components syntax', () => {
+  const Header = <PageHeader logo="Logo" headerTools="PageHeaderTools | Avatar" topNav="Navigation" />;
+  const Sidebar = <PageSidebar isNavOpen />;
+
+  const view = mount(
+    <Page {...props} header={Header} sidebar={Sidebar}>
+      <PageGroup sticky="bottom" hasShadowTop>
```

Comment:
You don't need to specify a shadow if it's sticky. A sticky position will automatically apply the correct shadow (top sticky = bottom shadow, bottom sticky = top shadow)

```suggestion
      <PageGroup sticky="bottom">
```